src/main/java/com/maxmind/minfraud/response/Device.java

```diff
@@ -0,0 +1,35 @@
+package com.maxmind.minfraud.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * This class contains minFraud response data related to the device.
+ */
+public final class Device {
+    private final String id;
+
+    public Device(
+            @JsonProperty("id") String id
+    ) {
+        this.id = id;
+    }
+
+    public Device() {
```

Comment:
I was mostly cutting and pasting here. Can I just move the `@JsonProperty` to the attribute declaration instead?